### PR TITLE
handle escape key while editing

### LINKED ISSUE #15: Escape key not working after editing
It seems like after editing content the keyboard shortcuts become unavailable, most importantly the escape key.

Ideally, after editing the user would hit the escape key to go back to 'presentation' mode where the keyboard shortcuts would become available again (space and arrow keys).

### PR DESCRIPTION
**Pull request by aaronpk**
while editing, handle the escape key and exit edit mode. closes #15.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -316,7 +316,11 @@ function stop_teleprompter()
 // Update Teleprompter
 function update_teleprompter()
 {
-	$.cookie('teleprompter_text', $('#teleprompter').html());
+	if(evt.keyCode == 27) { // escape
```
Comment (manifestinteractive) [inline js/script.js:319]: There is no `evt` param being passed into this function, so not sure how this condition could ever be true.

File: js/script.js [hunk 2]
```diff
@@ -316,7 +316,11 @@ function stop_teleprompter()
 // Update Teleprompter
 function update_teleprompter()
```
Comment (manifestinteractive) [inline js/script.js]: Probably need to update this to `update_teleprompter(evt)` otherwise the `evt` used inside this function will be undefined

**Review by manifestinteractive — CHANGES_REQUESTED** [review]
Seems like this will be helpful once we address the missing `evt` 👍

File: js/script.js (hunk 1 above)
Comment (aaronpk) [reply js/script.js:319]: You're right, that's what I get for trying to copy this out of my actual working copy!

**Comment by aaronpk** [comment]
Thanks for catching that! I'm working on a much larger change, but wanted to get this in, and made a mistake copying the code out of my working copy! I've added the missing param so it should work now!

File: js/script.js [hunk 3]
```diff
@@ -314,9 +314,13 @@ function stop_teleprompter()
 }
 
 // Update Teleprompter
-function update_teleprompter()
+function update_teleprompter(evt)
 {
-	$.cookie('teleprompter_text', $('#teleprompter').html());
+	if(evt.keyCode == 27) { // escape
```
Comment (manifestinteractive) [inline js/script.js:319]: OK, promise this is the last thing, but just noticed this...

Different browsers use different ways of support keyCode ... could we do something like this?

```
var key = evt.which || evt.keyCode || 0;

if(key == 27) {
```

Comment (manifestinteractive) [reply js/script.js:319]: https://unixpapa.com/js/key.html `3.2. Classic Values Returned on Character Events` for reference

File: js/script.js (hunk 1 above)
Comment (donniebreve) [inline js/script.js:319]: Missing a space between if and (

**Review by donniebreve — APPROVED** [review]

**Review by manifestinteractive — CHANGES_REQUESTED** [review]
This ticket is still waiting to have the key code issue addressed before it can get merged.  Was hoping the original author could have addressed this, but looks like it is stuck.  I will work on deploying my own fix if the requested changes are not addressed by end of week.

**Comment by manifestinteractive** [comment]
Declining this PR because of merge conflicts, however I am manually integrating a version of these changes in the next release ( v1.2.0 )

**Pull request closed without merging**